ChaosMod/Memory/Hooks/HandleToEntityStructHook.cpp

```diff
@@ -1,7 +1,6 @@
 #include <stdafx.h>
 
-static Entity m_origHandle = 0;
-static Entity m_proxyHandle = 0;
+static std::map<Entity, Entity> vehicleMap;
```

Comment:
A hash map (`unordered_map`) would be more fitting here. Also it'll need to be cleaned up regularly by removing all deleted entities from it to avoid it from growing infinitely.